packages/sirius-web/frontend/sirius-web-application/src/views/edit-project/workbench-views/representations/RepresentationsView.tsx

```diff
@@ -27,19 +29,46 @@ import {
   ListPropertySection,
   TreePropertySection,
 } from '@eclipse-sirius/sirius-components-forms';
+import SyncLockOutlinedIcon from '@mui/icons-material/SyncLockOutlined';
+import SyncOutlinedIcon from '@mui/icons-material/SyncOutlined';
+import Box from '@mui/material/Box';
+import Tooltip from '@mui/material/Tooltip';
 import Typography from '@mui/material/Typography';
-import { ForwardedRef, forwardRef, useEffect, useImperativeHandle, useState } from 'react';
+import { ForwardedRef, forwardRef, MutableRefObject, useEffect, useRef, useState } from 'react';
 import { makeStyles } from 'tss-react/mui';
+import { useSelectedElementsLabels } from '../useSelectedElementsLabels';
 import { RepresentationsViewState } from './RepresentationsView.types';
+import { useRepresentationsViewHandle } from './useRepresentationsViewHandle';
 import { useRepresentationsViewSubscription } from './useRepresentationsViewSubscription';
 import { GQLFormRefreshedEventPayload } from './useRepresentationsViewSubscription.types';
 
-const useRepresentationsViewStyles = makeStyles()((theme) => ({
+const useViewStyles = makeStyles()((theme) => ({
```

Comment:
Same comment. I suspect that this was changed from some copied / paster but it created the copy/paste of unused style in some part